ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -1,217 +1,211 @@
+{%         if ethernet_interfaces[ethernet_interface].ipv6_enable is arista.avd.defined(true) %}
    ipv6 enable
-{%             endif %}
-{%             if ethernet_interfaces[ethernet_interface].ipv6_address is arista.avd.defined %}
+{%         endif %}
+{%         if ethernet_interfaces[ethernet_interface].ipv6_address is arista.avd.defined %}
    ipv6 address {{ ethernet_interfaces[ethernet_interface].ipv6_address }}
-{%             endif %}
-{%             if ethernet_interfaces[ethernet_interface].ipv6_address_link_local is arista.avd.defined %}
+{%         endif %}
+{%         if ethernet_interfaces[ethernet_interface].ipv6_address_link_local is arista.avd.defined %}
    ipv6 address {{ ethernet_interfaces[ethernet_interface].ipv6_address_link_local }} link-local
-{%             endif %}
-{%             if ethernet_interfaces[ethernet_interface].ipv6_nd_ra_disabled is arista.avd.defined(true) %}
+{%         endif %}
+{%         if ethernet_interfaces[ethernet_interface].ipv6_nd_ra_disabled is arista.avd.defined(true) %}
    ipv6 nd ra disabled
-{%             endif %}
-{%             if ethernet_interfaces[ethernet_interface].ipv6_nd_managed_config_flag is arista.avd.defined(true) %}
+{%         endif %}
+{%         if ethernet_interfaces[ethernet_interface].ipv6_nd_managed_config_flag is arista.avd.defined(true) %}
    ipv6 nd managed-config-flag
-{%             endif %}
-{%             if ethernet_interfaces[ethernet_interface].ipv6_nd_prefixes is arista.avd.defined %}
-{%                 for prefix in ethernet_interfaces[ethernet_interface].ipv6_nd_prefixes %}
-{%                    set ipv6_nd_prefix_cli = "ipv6 nd prefix " ~ prefix %}
-{%                    if ethernet_interfaces[ethernet_interface].ipv6_nd_prefixes[prefix].valid_lifetime is arista.avd.defined %}
-{%                       set ipv6_nd_prefix_cli = ipv6_nd_prefix_cli ~ ethernet_interfaces[ethernet_interface].ipv6_nd_prefixes[prefix].valid_lifetime %}
-{%                    endif %}
-{%                    if ethernet_interfaces[ethernet_interface].ipv6_nd_prefixes[prefix].preferred_lifetime is arista.avd.defined %}
-{%                       set ipv6_nd_prefix_cli = ipv6_nd_prefix_cli ~ ethernet_interfaces[ethernet_interface].ipv6_nd_prefixes[prefix].preferred_lifetime %}
-{%                    endif %}
-{%                    if ethernet_interfaces[ethernet_interface].ipv6_nd_prefixes[prefix].no_autoconfig_flag is arista.avd.defined %}
-{%                       set ipv6_nd_prefix_cli = ipv6_nd_prefix_cli ~ " no-autoconfig" %}
-{%                    endif %}
+{%         endif %}
+{%         if ethernet_interfaces[ethernet_interface].ipv6_nd_prefixes is arista.avd.defined %}
+{%             for prefix in ethernet_interfaces[ethernet_interface].ipv6_nd_prefixes %}
+{%                 set ipv6_nd_prefix_cli = "ipv6 nd prefix " ~ prefix %}
+{%                 if ethernet_interfaces[ethernet_interface].ipv6_nd_prefixes[prefix].valid_lifetime is arista.avd.defined(true) %}
```

Comment:
these should not be true, since it is the value we want. same for line 110, but 113 is ok.

Reply:
I don't get that one. Will ping you

Reply:
understood